Migration.md

```diff
@@ -5,6 +5,35 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Ignition Rendering 5.x to 6.x
+
+### Modifications
+
+1. **Scene.hh**
+    + Added `Scene::PostRender`. The function `Camera::Render` must be executed
+      between calls to `Scene::PreRender` and `Scene::PostRender`. Failure to do
+      so will result in asserts triggering informing users to correct their code.
+      Alternatively calling `Scene::SetCameraPassCountPerGpuFlush( 0 )` avoids
+      this strict requirement.
+      Users handling only one Camera can call `Camera::Update` or `Camera::Capture`
+      and thus do not need to worry.
+      However for more than one camera (of any type) the optimum way to handle them is to update them in the following pattern:
+      ```
+      scene->PreRender();
+      for( auto& camera in cameras )
+      {
+          camera->PreRender();
```

Comment:
do we need to call `camera->PreRender()`? The original behavior does not need to do this as `scene->PreRender` goes through all the nodes and call `PreRender` for all cameras and visuals.

Looking at your branch in ign-sensors:
https://github.com/darksylinc/ign-sensors/blob/matias-PostFrame5/src/RenderingSensor.cc#L98-L120

I think it's still not needed right?

Reply:
I would have to check the code but I didn't touch Camera::PreRender I just assumed that was the behavior.

If you say it's not needed then it's probably not but I'll better double check

Reply:
Yes. You are correct.

I checked the code and it was an error on my part. `Camera::PreRender` is called implicitly by `Scene::PreRender` and the user does not have to call it again. Fortunately it doesn't seem like code was affected, just documentation.